packages/manager/apps/pci-workflow/src/pages/executions/Executions.page.spec.tsx

```diff
@@ -4,8 +4,21 @@ import * as useExecutionsHook from '@/api/hooks/useExecutions';
 import { wrapper } from '@/wrapperRenders';
 import Executions from './Executions.page';
 
-vi.mock('@ovhcloud/manager-components', async () => {
-  const mod = await vi.importActual('@ovhcloud/manager-components');
+vi.mock('@ovh-ux/manager-react-components', async () => {
+  const mod = await vi.importActual('@ovh-ux/manager-react-components');
+  return {
+    ...mod,
+    useDatagridSearchParams: () => ({
+      pagination: vi.fn(),
+      setPagination: vi.fn(),
+      sorting: vi.fn(),
+      setSorting: vi.fn(),
+    }),
+    useProjectUrl: () => 'project_url',
+  };
+});
```

Comment:
You can move this mock since it will be used repeatedly within the setupFile [feel free to include it now or leave it for another PR].